src/register/RegistrationPage.test.jsx

```diff
@@ -153,11 +148,7 @@ describe('RegistrationPage', () => {
   };
 
   const ssoProvider = {
```

Comment:
Do we still need to pass ssoProvider in `formPayload`? Reference line 216.

Reply:
Nope, we don't need this. Thanks for pointing this out.